packages/wallet/src/types/transaction.ts

```diff
@@ -0,0 +1,32 @@
+import { Amount } from './amount';
+
+export interface TransferTransaction {
```

Comment:
We have [an open PR](https://github.com/pactus-project/pactus/pull/1767) to support [Open-RPC](https://open-rpc.org/). 
Once it's merged, we can generate client code and publish a new package named `pacrus-rpc`, which will include both gRPC and JSON-RPC client code. Then, you won't need to declare them here.

Please help us proofread the PR and later generate the client code.